### DISCUSSION TITLE
Custom draw in timeline clip ?

### DISCUSSION THREAD

**Discussion by eddy16180** [AcademySoftwareFoundation/OpenTimelineIO#1656] [General]
Hi,

I'm new to OTIO and so far it has saved me a lot of time. Great work everyone involved !

I have a question in regards to custom draws of a clip. I was wondering if its possible to draw a custom overlay for a given clip.
Eg i'd like to draw a graph for each frame to visualize some additional properties for the user. In essence it would be similar to displaying the wave form of an audio track overlaid on the clip.

Apologies if this has been asked before or has been documented somewhere I didn't look so far.

Any hints would be appreciated.

Kind regards,
Christoph

**Comment by jminor** [comment]
It looks like no one responded to this - sorry about that! If you mean drawing an overlay in the otioview application, or Raven, then the source code is available and could be modified to do what you're asking about, but there isn't a feature in the tool itself or in the OTIO schema for that.
